wp-includes/plugin.php

```diff
@@ -208,7 +208,7 @@ function apply_filters( $tag, $value ) {
 
 	do {
 		foreach( (array) current($wp_filter[$tag]) as $the_ )
-			if ( !is_null($the_['function']) ){
+			if ( !is_null($the_['function']) && $the_['function'] != '' ){
```

Comment:
i don't remember  the exact use case  but now you ask the question '!empty()' seems a better solution.